LBBT-Design-Patterns/LBBTCalculator.cs

```diff
@@ -1,60 +1,66 @@
-﻿using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
-
-namespace LBBT_Design_Patterns
+﻿using System.Collections.Concurrent;
+
+namespace LBBT_Design_Patterns;
+
+/// <summary>
+///   Tax calculator using tax bands with rates.
+/// </summary>
+/// <param name="taxStrategies">
+///   Tax strategies to use in calculations,
+///   or will use default strategies specified in <see cref="CreateDefaultStrategies"/> if null.
+/// </param>
+public class LBBTCalculator(IList<ITaxStrategy>? taxStrategies = null) : ITaxCalculator
```

Comment:
Taking in strategies list as parameter enables flexibility or more complete testing. Set a default value of null so that you don't have to enter this parameter when calling and we can handle the null later